packages/app/src/debug/DebugSpec.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div
+    data-cy="debug-spec-col"
+    class="grid flex flex-col px-24px gap-24px self-stretch"
+  >
+    <div
+      data-cy="debug-spec-item"
+      class="w-full overflow-hidden flex flex-col items-start box-border border-t-1px border-x-1px rounded"
+    >
+      <div
+        data-cy="debug-spec-header"
+        class="w-full flex flex-row items-center py-12px bg-gray-50 border-b-1px border-b-gray-100 rounded-t"
+      >
+        <div
+          data-cy="spec-contents"
+          class="w-full flex flex-row px-16px items-center"
+        >
+          <div
+            data-cy="spec-path"
+            class="flex-grow non-italic text-base truncate"
+          >
+            <span
+              class="font-normal text-gray-600"
+            >
+              {{ specData.path }}
+            </span>
+            <span
+              class="inline-flex items-center"
+            >
+              <SpecNameDisplay
+                :spec-file-name="specData.fileName"
+                :spec-file-extension="specData.extension"
+              />
+            </span>
+          </div>
+          <div
+            type="button"
+            class="ml-10px"
+          >
+            <Button
+              data-cy="run-failures"
+              variant="white"
+              class="inline-flex gap-x-10px whitespace-nowrap justify-center items-center isolate"
+              :disabled="isDisabled"
+              :to="{ path: '/specs/runner', query: { file: (spec.path).replace(/\\/g, '/') } }"
+            >
+              <template #prefix>
+                <IconActionRefresh
+                  data-cy="icon-refresh"
+                  stroke-color="indigo-500"
+                />
+              </template>
+              Run Failures
+            </Button>
+          </div>
+        </div>
+      </div>
+      <ul
```

Comment:
Is this supposed to be `<ul>`? It does not contain any `<li>` - it seems odd to have a list without any list items.

Reply:
The `DebugFailedTest` used to render `li` elements, but no longer does.  This can just be a `div` now.